### PR TITLE
Add Option to set another include line

### PR DESCRIPTION
**Pull request by stevie-**
+FIX: check if omapi-key is not empty

### CODE REVIEW

**Comment by ekohl** [comment]
:+1:

File: templates/dhcpd.conf.erb [hunk 1]
```diff
@@ -84,3 +84,11 @@ filename "<%= @pxefilename %>";
 log-facility <%= @logfacility %>;
 
 include "<%= @dhcp_dir %>/dhcpd.hosts";
+
+<% if @includes.is_a? Array -%>
+<%   @includes.each do |include| -%>
```
Comment (mmoll) [inline templates/dhcpd.conf.erb]: too many spaces here

File: templates/dhcpd.conf.erb [hunk 2]
```diff
@@ -84,3 +84,11 @@ filename "<%= @pxefilename %>";
...
+include "<%= include %>";
+<%   end -%>
```
Comment (mmoll) [inline templates/dhcpd.conf.erb]: and here

**Comment by mmoll** [comment]
:+1: about the code as such. A spec test would be nice...
One question: what's your specific use case? becuase by using includes we're opening up the possibility to pull in things outside of puppet control into the dhcpd config, which makes me feel uneasy (but I'm also using it for pulling in Sun Ray and iPXE information).

**Comment by stevie-** [comment]
That's my usecase. We have another process which creates host entrys in a extra file. 
So we can use both.

File: templates/dhcpd.conf.erb (hunk 2 above)
Comment (stevie-) [inline templates/dhcpd.conf.erb]: Copy paste from above code. I was wondering as well.

**Comment by mmoll** [comment]
OK, while I think this is ready to go, please squash your commits together into one commit and, if possible, extend the spec tests (extending the one for the default case and adding one with includes set).

**Comment by mmoll** [comment]
@stevie- ping?

**Comment by ekohl** [comment]
@stevie- FYI: github doesn't send a notification when you push so we didn't see your update. I've pushed this change including tests. Thanks!

**Pull request closed without merging**